addons/attributes/stringtable.xml

```diff
@@ -17,6 +17,18 @@
             <Chinesesimp>Zeus Enhanced - 属性</Chinesesimp>
             <Turkish>Zeus Enhanced - Özellikler</Turkish>
         </Key>
+        <Key ID="STR_ZEN_Attributes_AmmoCargo">
+            <English>Ammo Cargo</English>
+        </Key>
+        <Key ID="STR_ZEN_Attributes_FuelCargo">
+            <English>Fuel Cargo</English>
+        </Key>
+        <Key ID="STR_ZEN_Attributes_RepairCargo">
+            <English>Repair Cargo</English>
+        </Key>
+        <Key ID="STR_ZEN_Attributes_SupportCargo_Tooltip">
+            <English>Cargo to supply to other vehicles.</English>
```

Comment:
Is this tooltip necessary?

Reply:
Can certainly be removed.

Reply:
It sounds a bit too generic. I guess we can either remove or add more specific tooltips.